apps/backoffice-v2/src/pages/Entities/components/CaseCreation/CaseCreation.tsx

```diff
@@ -28,7 +28,11 @@ export const CaseCreation = withCaseCreation(() => {
           <span>Add case manually</span>
         </Button>
       </SheetTrigger>
-      <SheetContent side="right" style={{ right: 0, top: 0 }} className="max-w-[620px]">
+      <SheetContent
+        side="right"
+        style={{ right: 0, top: 0 }}
+        className="max-w-[620px] overflow-y-auto"
```

Comment:
Adding scroll to the sheet

Reply:
Proud of you!

Reply:
There is a PR open that uses the scroll area component which has a design compared to the native browser overflow.